src/server.ts

```diff
@@ -24,11 +21,14 @@ import {
  * environment/platform-specific values through to your loader/action.
  */
 export type GetLoadContextFunction = (
-  request: FastifyRequest,
-  reply: FastifyReply
+  req: FastifyRequest,
```

Comment:
in fastify land, it seems pretty common to use `request` and `reply` instead of `req` and `res`

https://www.fastify.io/docs/latest/Guides/Getting-Started/

Reply:
changed it, I prefer a short versions :)